ruby/ql/lib/codeql/ruby/StringOps.qll

```diff
@@ -0,0 +1,180 @@
+/**
+ * Provides classes and predicates for reasoning about string-manipulating expressions.
+ */
+
+private import ruby
+private import codeql.ruby.DataFlow
+private import codeql.ruby.controlflow.CfgNodes
+private import InclusionTests
+
+/**
+ * Provides classes for reasoning about string-manipulating expressions.
+ */
+module StringOps {
```

Comment:
Is this module wrapper needed?

Reply:
No, this just matches what JavaScript does.